crates/swc_ecma_transforms_typescript/src/config.rs

```diff
@@ -70,3 +90,7 @@ pub enum ImportsNotUsedAsValues {
 
 #[deprecated = "ImportNotUsedAsValues is renamed to ImportsNotUsedAsValues"]
 pub type ImportNotUsedAsValues = ImportsNotUsedAsValues;
+
+const fn true_by_default() -> bool {
```

Comment:
We have to use create a `true_by_default` for `ts_enum_is_readonly`.
Is setting `ts_enum_is_mutable: false` preferable?

Reply:
I think we need `derive(SerdeDefault)` 🤣 
I'll create it soon.

Reply:
I think so. It would allow me to revert a decent amount of this code. @kdy1 does that sound good?

Reply:
Yes, let's do it